src/tools/pontoon/ui.md

```diff
@@ -4,115 +4,125 @@
 
-![Profile menu](../../assets/images/pontoon/ui/profile_menu.png)
+### String list and filters
 
-Note that some profile menu items are restricted by [user rights](users.md#user-roles).
+The sidebar displays the list of strings in the current project resource. For each string there is:
+* A colored square that identifies the string status (i.e. *Missing*, *Translated*, etc.).
+* The source string.
+* The approved translation or the most recent suggestion if available.
 
-### Sidebar
-
-The sidebar displays the list of strings in the current project resource. Each string is displayed with the string status (i.e. *Missing*, *Translated*, etc.) identified by a colored square, the source string, and the approved translation or the most recent suggestion if available.
-
-![Sidebar](../../assets/images/pontoon/ui/sidebar.png)
+![Sidebar](../../assets/images/pontoon/ui/sidebar.png "Screenshot of the sidebar, with a list of strings showing the different string statuses")
 
 Color legend:
 * **<span style="color: #7bc876;">green</span>**: string is translated.
-* **<span style="color: #fed271;">yellow</span>**: string is fuzzy.
+* **<span style="color: #c0ff00;">light-green</span>**: string is pretranslated.
 * **<span style="color: #ffa10f;">orange</span>**: string has warnings.
 * **<span style="color: #f36;">red</span>**: string has errors.
 * **<span style="color: #4d5967;">gray</span>**: string is missing.
 
-At the top of the sidebar, the translator can access Pontoon’s [search and string filters](search_filters.md).
+At the top of the sidebar, the user can access Pontoon’s [search and string filters](search_filters.md).
 
-Note that the sidebar toggles between the strings and the editing space when using the in-context editor.
+When a string is selected in the sidebar, a small icon with 4 arrows is displayed near the checkbox: this can be used to show strings that surround the selected string in the [resource](glossary.md#resource), bypassing the current filter. This is often helpful to provide more context for the localization, especially when translating missing strings.
+
+![Button to show sibling strings in sidebar](../../assets/images/pontoon/ui/sidebar_expand.png "Screenshot of the button to show sibling strings in sidebar")
 
 ### Translation workspace
 
-The translation workspace is where strings are translated.
+The translation workspace is where the translation takes place.
+
+![Translation workspace](../../assets/images/pontoon/ui/workspace_standard.png "Screenshot of the standard editor in the translation workspace")
 
-![Translation workspace](../../assets/images/pontoon/ui/workspace_standard.png)
+It’s possible to navigate sequentially through the strings by clicking the `PREVIOUS` or `NEXT` buttons located at the top of the workspace, or by using keyboard shortcuts (`ALT` + arrow down or up). The `COPY LINK` button copies in the clipboard a link to the current string.
```

Comment:
Is "The `COPY LINK` button copies a link to the current string to the clipboard." a better word order?

Reply:
My brain struggles more with to…to… in your version.

Reply:
Yeah...

Still to-to, but maybe sounding less weird:
A link to the current string can be copied to clipboard using the `COPY LINK` button.

I'm sure @bcolsson has better ideas. :)

Reply:
I can give it a shot:
The `COPY LINK` button adds the URL for the current string to your clipboard.

Reply:
I think it's useful to maintain "copy" (also, nit-picking you replace content in the clipboard, it's not a stack).

I'll go with Matjaz's latest suggestion.